products/angularjs.md

```diff
@@ -40,7 +40,7 @@ releases:
 -   releaseCycle: "1.6"
     releaseDate: 2016-12-08
     eol: 2021-12-31
-    extendedSupport: false
+    extendedSupport: true
```

Comment:
I think mixing different extended support offers in the same column is a bit confusing. @endoflife-date/everyone, how should we address this ?

Reply:
How about a custom column called ELTS Provider, with either of "HeroDevs", "HeroDevs, Perforce", or "Perforce".

Reply:
That sounds good, thanks

Reply:
Done.